package.json

```diff
@@ -53,15 +53,15 @@
     "base64url": "^3.0.1",
     "blipp": "^4.0.1",
     "camelcase": "^5.3.1",
-    "commander": "^2.20.0",
+    "commander": "^2.20.3",
     "csv-parse": "^2.4.0",
     "deep-map": "^1.5.0",
     "dotenv": "^4.0.0",
     "es6-error": "^4.1.1",
     "firstline": "^2.0.2",
     "good": "^8.1.2",
     "good-winston": "^4.0.0",
-    "handlebars": "^4.5.2",
+    "handlebars": "^4.5.3",
```

Comment:
Is handlebars actually used in the project?  I had a quick look and could only see this reference to it
Likewise `handlebars-helper-equal`